doandroidlib/src/main/java/in/tosc/doandroidlib/api/DigitalOceanClient.java

```diff
@@ -49,14 +53,9 @@
             @Field("name") String name              // for rename or snapshot
             );
 
+    //For getting regions
+    @GET("regions")
+    Call<Regions> getRegions();
 
-    @FormUrlEncoded
-    @POST("droplets")
-    Call<String> createDroplet(
-            @Field("name") String name,
-            @Field("region") String region,
-            @Field("size") String size,
-            @Field("image") String image
-    );
 
```

Comment:
@ashu10832 why have you deleted this api call ? it will later used to send a request to create droplet